javaslang/src/test/java/javaslang/collection/euler/Euler67Test.java

```diff
@@ -58,7 +59,7 @@ private static int solve(String fileName) {
         return Vector.ofAll(
                 Utils.readLines(Utils.file(fileName)).map(line ->
                         Arrays.stream(line.split("\\s")).map(Integer::parseInt)
-                ).map(s -> Vector.ofAll(s::iterator))
+                ).map(s -> List.ofAll(s::iterator).toVector())
```

Comment:
@paploric you are right - please keep the asTraversableAgain method then but please write a TODO that we remove it with 3.0.0. I think the test can be reverted then, right?